test/fixtures/long-running/jobs.yml.erb

```diff
@@ -1,3 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: multi-replica
+spec:
+  ports:
+  - port: 80
+    name: http
```

Comment:
Why does something that emulates a long-running background queue jobs need http spec?